packages/catalog-consumer/src/model/converter.ts

```diff
@@ -58,18 +58,20 @@ export const fromEServiceTechnologyV1 = (
 
 export const fromEServiceAttributeV1 = (
   input: EServiceAttributeV1
-): Attribute =>
-  match<EServiceAttributeV1, Attribute>(input)
+): Attribute[] =>
+  match<EServiceAttributeV1, Attribute[]>(input)
     .with(
       {
         single: P.not(P.nullish),
       },
-      ({ single }) => ({ id: single, ids: undefined })
+      ({ single }) => [
+        {
+          id: single.id,
+          explicitAttributeVerification: single.explicitAttributeVerification,
+        },
+      ]
```

Comment:
```suggestion
      ({ single }) => [single]
```
(I see in the `.otherwise` you're not remapping the two fields)